common/src/main/java/com/linecorp/centraldogma/common/PathPattern.java

```diff
@@ -50,6 +58,14 @@ static PathPattern of(String... patterns) {
         return of(ImmutableSet.copyOf(requireNonNull(patterns, "patterns")));
     }
 
+    /**
+     * Creates a path pattern with the {@code pathPatterns}.
+     */
+    static PathPattern of(PathPattern... pathPatterns) {
+        requireNonNull(pathPatterns, "pathPatterns");
```

Comment:
How about:
- throwing an IAE if `pathPatterns.length == 0`; and
- returning `pathPatterns[0]` if `pathPatterns.length == 1`?